tests/integration/targets/fusion_nig/tasks/main.yml

```
@@ -0,0 +1,36 @@
+- name: Create new network interface group
+  purestorage.fusion.fusion_nig:
+    name: "foo_group"
+    availability_zone: az1
+    region: pure-us-west
+    state: present  # or absent
```

Nit: redundant comment 
```suggestion
    state: present 
```